GG/src/DropDownList.cpp

```diff
@@ -185,6 +192,13 @@ void ModalListPicker::EndRun() {
 bool ModalListPicker::Dropped() const
 { return m_dropped; }
 
+void ModalListPicker::Render()
+{
+    // If the app changes size while the drop down is open, then close it.
+    if (m_initial_appsize != Pt(GG::GUI::GetGUI()->AppWidth(), GG::GUI::GetGUI()->AppHeight()))
```

Comment:
Is this preferable to connecting HumanClientApp::WindowResizedSignal to EndRun ?